### ISSUE TITLE
update (enhancements/question/observation)

### ISSUE THREAD

**Issue by tgtje** [e107inc/e107#608]
Sorry to barge in this way, just tested with the latest files available (29-5)
Update 'seems' to get along, but event calender and forum have (for me) most troublesome issues.
Now i am (with testing a lot) at a point to have to mention, that the probability is going to exist that full FRESH install files and Updating sites can NOT use (all) the same  files.

I can post a lot of remarks and why and how (imo)..i would propose a discussion first, since updating seems to go another path then a fresh install !! .

NOT MEANT negative !! observations!! (that is why testing is done...)

Basically it comes to > as long as we think  = user!! = english.. will be allright for the most (there are still things to be considered  # ht access renaming, unadjusted config file), but as soon as other (pre)installed languages are in use......that is another story..
